### PR TITLE
chore(deps): remove dependency @types/react-helmet

### PR DESCRIPTION
**Pull request by Yeti-or**
:baby_chick: Download canary assets:

[colors--canary.424.4460410092.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.424.4460410092.xml)  
[color_metro_ios-swift--canary.424.4460410092.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.424.4460410092.swift)  
[color_metro_kotlin--canary.424.4460410092.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.424.4460410092.kt)  
[color_metro_react-native--canary.424.4460410092.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.424.4460410092.ts)  
[color_sberHealth_ios-swift--canary.424.4460410092.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.424.4460410092.swift)  
[color_sberHealth_kotlin--canary.424.4460410092.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.424.4460410092.kt)  
[color_sberHealth_react-native--canary.424.4460410092.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.424.4460410092.ts)  
[color_sbermarket_ios-swift--canary.424.4460410092.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.424.4460410092.swift)  
[color_sbermarket_kotlin--canary.424.4460410092.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.424.4460410092.kt)  
[color_sbermarket_react-native--canary.424.4460410092.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.424.4460410092.ts)  
[color_sberprime_ios-swift--canary.424.4460410092.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.424.4460410092.swift)  
[color_sberprime_kotlin--canary.424.4460410092.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.424.4460410092.kt)  
[color_sberprime_react-native--canary.424.4460410092.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.424.4460410092.ts)  
[color_selgros_ios-swift--canary.424.4460410092.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.424.4460410092.swift)  
[color_selgros_kotlin--canary.424.4460410092.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.424.4460410092.kt)  
[color_selgros_react-native--canary.424.4460410092.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.424.4460410092.ts)  
[color_smbusiness_ios-swift--canary.424.4460410092.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.424.4460410092.swift)  
[color_smbusiness_kotlin--canary.424.4460410092.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.424.4460410092.kt)  
[color_smbusiness_react-native--canary.424.4460410092.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.424.4460410092.ts)  
[PlasmaTokensColor--canary.424.4460410092.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.424.4460410092.swift)  
[shadow_sbermarket_react-native--canary.424.4460410092.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.424.4460410092.ts)  
[typo_mage_ios-swift--canary.424.4460410092.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.424.4460410092.swift)  
[typo_mage_kotlin--canary.424.4460410092.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.424.4460410092.kt)  
[typo_mage_react-native--canary.424.4460410092.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.424.4460410092.ts)  
[typo_plasma_ios-swift--canary.424.4460410092.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.424.4460410092.swift)  
[typo_plasma_kotlin--canary.424.4460410092.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.424.4460410092.kt)  
[typo_plasma_react-native--canary.424.4460410092.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.424.4460410092.ts)  
[typo_ruler_ios-swift--canary.424.4460410092.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.424.4460410092.swift)  
[typo_ruler_kotlin--canary.424.4460410092.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.424.4460410092.kt)  
[typo_ruler_react-native--canary.424.4460410092.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.424.4460410092.ts)  
[typo_sage_ios-swift--canary.424.4460410092.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.424.4460410092.swift)  
[typo_sage_kotlin--canary.424.4460410092.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.424.4460410092.kt)  
[typo_sage_react-native--canary.424.4460410092.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.424.4460410092.ts)  
[typo_sbermarket_ios-swift--canary.424.4460410092.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.424.4460410092.swift)  
[typo_sbermarket_kotlin--canary.424.4460410092.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.424.4460410092.kt)  
[typo_sbermarket_react-native--canary.424.4460410092.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.424.4460410092.ts)  
[typo_soulmate_ios-swift--canary.424.4460410092.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.424.4460410092.swift)  
[typo_soulmate_kotlin--canary.424.4460410092.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.424.4460410092.kt)  
[typo_soulmate_react-native--canary.424.4460410092.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.424.4460410092.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.163.1-canary.424.4460410092.0
  npm install @salutejs/plasma-temple@1.144.1-canary.424.4460410092.0
  # or 
  yarn add @salutejs/plasma-b2c@1.163.1-canary.424.4460410092.0
  yarn add @salutejs/plasma-temple@1.144.1-canary.424.4460410092.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-424/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-424/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-424/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-424/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-424/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-424/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-424/temple-storybook/

File: website/plasma-temple-docs/package.json [hunk 1]
```diff
@@ -41,22 +41,13 @@
...
-            "last 1 safari version"
-        ]
+        "production": [">0.5%", "not dead", "not op_mini all"],
+        "development": ["last 1 chrome version", "last 1 firefox version", "last 1 safari version"]
     },
     "devDependencies": {
         "@docusaurus/module-type-aliases": "^2.0.0-beta.4",
         "@tsconfig/docusaurus": "^1.0.3",
         "@types/react": "^17.0.17",
-        "@types/react-helmet": "^6.1.2",
```
Comment (Yakutoc) [inline website/plasma-temple-docs/package.json:59]: А почему решили удалить?

**Review by Yakutoc — APPROVED** [review]

File: website/plasma-temple-docs/package.json (hunk 1 above)
Comment (Yeti-or) [reply website/plasma-temple-docs/package.json:59]: потому что пакет `react-helmet` не используется @neretin-trike подтверди плиз

Comment (neretin-trike) [reply website/plasma-temple-docs/package.json:59]: ну да, я тоже его не вижу

Comment (neretin-trike) [reply website/plasma-temple-docs/package.json:59]: ну вообще он есть как зависимость в `@docusaurus/core`. Но самих компонент вроде Helmet нет

**Review by neretin-trike — APPROVED** [review]

**Pull request merged**